test/testother.cpp

```diff
@@ -3371,6 +3374,15 @@ class TestOther : public TestFixture {
         ASSERT_EQUALS("[test.cpp:1]: (style) Parameter 's1' can be declared as reference to const\n"
                       "[test.cpp:1]: (style) Parameter 's2' can be declared as reference to const\n",
                       errout.str());
+
+        check("struct S {\n"
+              "    void f(int& r) { p = &r; }\n"
```

Comment:
I think `isVariableChanged` can be updated to handle this case.

Reply:
I would not even expect `isVariableChanged()` to return `true` here.